### ISSUE TITLE
Robocop Format & Check Conflicts for line-too-long

### ISSUE THREAD

**Issue by fcolak** [MarketSquare/robotframework-robocop#1515]
In issue #1444, a similar problem is resolved but not fully.

I do `robocop format` & `robocop check` and they are conflicting with each other.

```
$ pip freeze | grep -i robocop  
robotframework-robocop==6.10.1
```

`robocop format` converts this line to:
```
VAR    ${test}  
...    verylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglineline  # robocop: off=line-too-long
```

```
VAR    ${test}  # robocop: off=line-too-long
...    verylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglineline
```

And `robocop check` now complains:

```
MISC15 Disabler directive found for 'line-too-long' rule(s) but no violation found

 108 |                 VAR    ${test}  # robocop: off=line-too-long
     |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^ MISC15
 109 |                 ...    verylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglineline

LEN08 Line is too long (151/120)
 
 108 |                 VAR    ${test}  # robocop: off=line-too-long
 109 |                 ...    verylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglinelineverylonglinelonglinglonglineline
     |                                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ LEN08
```

Similar case is happening with other keywords; such as `Append To List`

**Comment by bhirsz** [comment]
Changes will be released in the 7.0.0 (during next week). In the end I redesigned our disablers a bit. Robocop disablers disable either whole blocks or specific lines. After the change they will always apply to whole node or blocks.

For example:

```
FOR    ${var}    IN RANGE   10  # robocop: off

END

Keyword  # robocop: off
...    ${arg}
...    ${arg2}
```

Previously ``# robocop: off`` applied only to specific line (like FOR loop header or first line of keyword call) but now it applies to whole node. It shall address your issue. There is small chance that some people will want to disable only specific line (ie. ignore issue only in the header, not whole keyword / for etc.) but I believe current approach is still better than the previous one.
